.yarn/patches/@metamask-network-controller-npm-19.0.0-a5e0d1fe14.patch

```diff
@@ -1,24 +1,4263 @@
+diff --git a/dist/NetworkController.js b/dist/NetworkController.js
```

Comment:
Is it possible to just release the fixed network-controller package really quickly and then update to it?

Reply:
i don't think that'd be realistically possible given the breaking change in 20.0.0 removes providerConfig from state. This is probably why we are still on 19.0.0

https://github.com/MetaMask/core/blob/main/packages/network-controller/CHANGELOG.md#removed


I'm going to guess that you're suggesting this since the patch is not human readable

Reply:
> Is it possible to just release the fixed network-controller package really quickly and then update to it?

It is not possible to do this in a timely manner, but we need to get this fix into prod asap